README.md

```diff
@@ -55,9 +55,109 @@ $ yarn audit
 ```shell
 $ yarn audit:fix
 ```
+### Setup AWS SSO Login
+```shell
+$ aws configure sso
+```
+- Choose session name: _{enter your name}_
+- Accept authorisation in browser
+- Choose the AWS account in the dropdown
+- CLI default client Region: eu-west-2
+- CLI default output format: json
+- aws sso login --profile _{profile name provided for account}
 
-## Additional Info
+## Architecture Diagram
+<img src="img.png" alt="img.png" width="500"/>
+
+## How To Deploy
+### 1. Setup Base CloudFormation Stacks
+To deploy the base common CloudFormation stacks required created by Dev Platform prior to deploying the solution use the Stack Orchestation tool provided in our [stack-orchestration](stack-orchestration) directory and deploy the [production_bootstrap.sh](stack-orchestration%2Fproduction_bootstrap.sh) script.
+
+***The stacks to be deployed are:***
+1. alerting-integration
+2. api-gateway-logs
+3. certificate-expiry
+4. vpc
+5. lambda-audit-hook
+6. checkov-hook
+7. infra-audit-hook
+
+#### Stack Orchestation Tool Prerequisites
```

Comment:
since the stack orchestration tool is being referenced in several places in this readme, I think it would be clearer to have a section just dedicated to it before you start explain how to deploy stuff.